pkgs/applications/science/math/scilab-bin/default.nix

```diff
@@ -65,6 +65,31 @@ stdenv.mkDerivation rec {
     ln -s "$out/opt/scilab-${ver}/bin/scilab" "$out/bin/scilab-${ver}"
     ln -s "$out/opt/scilab-${ver}/bin/scilab" "$out/bin/scilab"
     ln -s "scilab-${ver}" "$out/bin/scilab-${majorVer}"
+
+    mkdir -p "$out/share/applications"
+    mv "$out/opt/scilab-${ver}/share/applications/scilab-adv-cli.desktop" "$out/share/applications"
```

Comment:
Can we not do `mv $out/opt/scilab-${ver}/share/applications/*.desktop $out/share/applications`?

Reply:
Not a bad idea, that's exactly what I started out with, but it (the error from mv) said no file was found.

Reply:
That could have been because of the quoting.

Reply:
Ah, you were right, thanks!